pkgs/test_core/lib/src/runner/configuration.dart

```diff
@@ -75,6 +75,11 @@ class Configuration {
   bool get branchCoverage => _branchCoverage ?? false;
   final bool? _branchCoverage;
 
+  /// A list of regular expressions to match against the package names during
+  /// coverage collection. Only packages that match at least one of the regexps
```

Comment:
[nit] add a blank line between header sentence and the details.

https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph